Games/types/Mafia/roles/cards/CryOutVisitors.js

```diff
@@ -0,0 +1,29 @@
+const Card = require("../../Card");
+const { PRIORITY_CRY_VISITORS } = require("../../const/Priority");
+
+module.exports = class CryOutVisitors extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.actions = [
+            {
+                priority: PRIORITY_CRY_VISITORS,
+                labels: ["hidden", "absolute"],
+                run: function () {
+                    if (this.game.getStateName() != "Night")
+                        return;
+
+                    let visitors = this.actor.role.data.visitors;
+                    let names = visitors?.map(visitor => visitor.name);
+
+                    if (visitors) {
```

Comment:
An empty array isn't a falsy value in Javascript. You need to do `if(visitors.length) `

or `if(visitors?.length)` if you want to handle null/undefined